lemonde.fr.txt

```diff
@@ -8,6 +8,8 @@ date: //time[@itemprop='dateModified']/@datetime
 # Publication date
 date: //time[@itemprop='datePublished']/@datetime
 
+# Remove "Lire aussi" blocks
+strip: //p[@class='lire']
```

Comment:
Yes, it is clearly an aesthetic choice. However, I think that (in the context of Wallabag), what you want is the content of the article you see to read it later. If you want links to other things, you directly go see them and add them too to wallabag.

However, I won't be upset if this particular line is not kept; I will just continue to add it in my files :)